uber/configspec.ini

```diff
@@ -1431,8 +1431,8 @@ email_post_con = boolean(default=False)
 1 = string(default="signups/placeholder_item.html")
 2 = string(default="signups/shirt_item.html")
 3 = string(default="signups/food_item.html")
-4 = string(default="signups/shifts_item.html")
 6 = string(default="signups/volunteer_agreement_item.html")
+99 = string(default="signups/shifts_item.html")
```

Comment:
@binary1230 what do you think of this idea to make sure "signups/shifts_item.html" is _always_ last? I know this means we'll have to change it _everywhere_ (including ubersystem-puppet and production-config)